adafruit_bitmap_font/ttf.py

```diff
@@ -6,19 +6,20 @@
 # Remove the above when TTF is actually supported.
 
 import struct
-
+from io import FileIO
+from displayio import Bitmap
 
 # https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6glyf.html
 
 
 class TTF:
-    def __init__(self, f, bitmap):
+    def __init__(self, f: FileIO, bitmap: Bitmap) -> None:
         f.seek(0)
         self.file = f
 
         self.characters = {}
 
-        def read(format):
+        def read(format: str) -> tuple:
```

Comment:
Same thing here, use `Tuple` from `typing`, but no worries on defining it any further.